content/en/service_management/service_level_objectives/monitor.md

```diff
@@ -31,24 +31,20 @@ Datadog monitor-based SLOs support the following monitor types:
 
 ## Setup
 
-On the [SLO status page][2], select **New SLO**.
-
-Under **Define the source**, select **Monitor Based**.
+On the [SLO status page][2], select **New SLO +**. Then select **By Monitor Uptime**.
 
 ### Define queries
 
-In the search box, start typing the name of a monitor. A list of matching monitors appears. Click on a monitor name to add it to the source list.
-
-If you're only using a single multi alert monitor in an SLO, you can optionally select "Calculate on selected groups" and pick up to 20 groups. Group selection is not supported for SLOs that contain multiple monitors. For SLOs with multiple monitors, you can add up to 20 monitors.
+In the search box, start typing the name of a monitor. A list of matching monitors appears. Click on a monitor name to add it to the source list:
```

Comment:
```suggestion
```suggestion
In the search box, start typing the name of a monitor to see a list of matching monitors. Click on a monitor name to add it to the source list.

**Notes**:
```
```